doc/usecases/record_elektra/UC_ansible_specification.md

```diff
@@ -0,0 +1,27 @@
+# Use Case: Specification of how a mount point shall be transformed into an Ansible task
+
+## Summary
+
+- **Title:** Specification of how a mount point shall be transformed into an Ansible task
+- **Scope:** Configuration
+- **Level:** User Goal
+- **Actors:** User (usually a sysadmin)
+- **Brief:** Based on [Ansible Export](UC_ansible_export.md), it should be possible to describe how keys in Elektra are transformed into tasks in an Ansible playbook.
+
+## Scenarios
+
+- **Precondition:** -
+- **Main success scenario:** 
+  - The user defines (e.g. using meta keys) how to translate keys in a mountpoint into a task in an Ansible playbook
+  - The user exports data of that mountpoints into an Ansible playbook
```

Comment:
Is this export also connected to UC_ansible_export.md?

Reply:
Yes! The idea of this use case was/is that we could (sometimes in the future) generate "real" Ansible playbooks that work without ansible-libelektra. But I think this is something out of scope for now.